src/manage/dashboard/index.md

```diff
@@ -1,71 +1,110 @@
 # Home Page and Dashboard
 [[toc]]
 
-When you log in to <AppDomainNameLink desc="Mergin Maps"/>, you will be presented with a dashboard for quick access to your recent active projects. Here, you can also manage your projects, [workspaces](../workspaces/), profile and [subscriptions](../subscriptions/).
+When you log in to <DashboardLink />, you will be presented with a quick overview of requests and recent active projects in your current workspace. Here, you can also manage your projects, [workspaces](../workspaces/), profile and [subscriptions](../subscriptions/).
 
-![Mergin Maps web dashboard](./web-dashboard-panel.jpg "Mergin Maps web dashboard")
+![Mergin Maps dashboard](./mergin-maps-dashboard.jpg "Mergin Maps dashboard")
 
 Current workspace and your username are displayed in the upper right corner of the screen.
```

Comment:
Edit text:
The current workspace and your username are displayed in the upper right corner of the screen.